.gitlab-ci.yml

```diff
@@ -0,0 +1,41 @@
+build:
+  stage: build
+  image: ghcr.io/acts-project/ubuntu2004_exatrkx:v17
+  artifacts:
+    paths:
+      - build/
+      - src/Examples/Python
+      - src/Examples/Scripts/Python
+      - src/Examples/Algorithms/TrackFinding/share
+      - src/Examples/Algorithms/Digitization/share
+    exclude:
+      - build/**/*.o
```

Comment:
This works, another option would be to add an install step and package up the install directory, and then running from that.

Up to you though.